src/doc/cli_manual/attributes.rst

```diff
@@ -22,17 +22,12 @@ these attributes is printed in italic.
 ----------------------------------------------
 |
 
-    +-----------------------+----------------------------+
-    | **Attribute**         | **Default/Allowed Values** |
-    +-----------------------+----------------------------+
-    | CreateCellsOfType     | DualGridAndStitchCells     |
-    +-----------------------+----------------------------+
-    |                       | *DualGridAndStitchCells*   |
-    +-----------------------+----------------------------+
-    |                       | *DualGrid*                 |
-    +-----------------------+----------------------------+
-    |                       | *StitchCells*              |
-    +-----------------------+----------------------------+
+    +-----------------------+------------------------------+
+    | **Attribute**         | **Default/Allowed Values**   |
```

Comment:
I fixed this in the sphinx extractor

Reply:
Thanks!

Another issue just occurred to me. Sorry I didn't see this earlier. Sphinx supports a few different ways of doing tables. This `.rst` output is currently using *grid* tables. Those are a pain to handle because everything needs to line up. As members change, spacing changes. So, its a pain to maintain manually. I would recommend adjusting the extractor to switch these to *[simple tables](http://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#tables)* or, perhaps just a normal *[list](http://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#lists-and-quote-like-blocks)* or *definition list*. IMHO, this would make maintaining this `.rst` code much easier for humans later. Other places in the current generation logic should be audited for use of *grid* tables too.

Reply:
Yes, I agree that grid tables are a pain to maintain. I did not know about the other options. I'll look into these and see what I can do.

Reply:
Just a quick question here: most of the tables are in the attributes.rst and events.rst files, which I have been assuming will be generated from the functions.rst file--correct me if I'm wrong. If this is the case, then the tables in attributes.rst and events.rst can remain grid tables because they won't be manually maintained. There are only about 5 tables in functions.rst, which I can manually change to simple tables in the next phase.

Reply:
Oh, I hadn't considered that. That sounds like a great idea. Or, maybe we keep the parts of the cli_extractor that auto-gen `attributes.rst` and `events.rst` so the process looks more like the following...

1. use (modified) cli_extrator.py to gen `attributes.rst` and `events.rst`
  1. In fact, all attributes objects are auto-gen'd from XML files, right? I dunno how often and/or even if attributes objects have english language prose documenting them but if they do, the `xml` file they are generated from would be the right/best place for that I think.
1. use new python script to create `MethodDoc.C` from `functions.rst`

The only place humans edit for documentation purposes is `functions.rst`.

That makes sense.